### ISSUE TITLE
kaboom examples and replit kaboom project no longer work in Safari 14.1

### ISSUE THREAD

**Issue by buffalox2** [replit/kaboom#42]
Seems like the recent updates to the kaboom library have changed something that has impacted Safari support. Previously all of the examples worked and I could learn to code using the kaboom library with my replit projects on Safari on a Mac or on my iPad. Now none of the examples work and my replit project is no longer working. This seems to have changed with the recent 0.5.0 release? The error message in Safari's web inspector is not very helpful to me but perhaps you understand it:

```
[Error] Error: 

	f (kaboom.js:1:9800)
	(anonymous function) (kaboom.js:1:7877)
	Et (kaboom.js:1:8678)
	(anonymous function) (kaboom.js:54:28043)
	Global Code (about:srcdoc:30)
```

The examples seem to work in the latest version of Google Chrome but as I can not use Chrome on my school issue iPads it would be better if Safari was supported.

**Comment by slmjkdbtl** [comment]
Yeah I've noticed there's some regression on Safari, this is high priority will investigate soon!

**Comment by slmjkdbtl** [comment]
looks like it's caused by bd13b80

**Comment by slmjkdbtl** [comment]
looks like it's caused by 51232b2

**Comment by slmjkdbtl** [comment]
fixed by 6c4f2eb

**Comment by buffalox2** [comment]
Thanks for fixing it!
